monitoring/understanding-the-monitoring-stack.adoc

```diff
@@ -9,18 +9,26 @@ toc::[]
 include::modules/monitoring-about-cluster-monitoring.adoc[leveloffset=+1]
 
 // Understanding the monitoring stack
-include::modules/monitoring-understanding-the-monitoring-stack.adoc[leveloffset=+1]
+include::modules/about-monitoring-stack-components.adoc[leveloffset=+1]
 include::modules/monitoring-default-monitoring-components.adoc[leveloffset=+2]
 include::modules/monitoring-default-monitoring-targets.adoc[leveloffset=+2]
 include::modules/monitoring-components-for-monitoring-user-defined-projects.adoc[leveloffset=+2]
+
+A cluster administrator can xref:../monitoring/configuring-the-monitoring-stack.adoc#configuring-the-monitoring-stack[configure the monitoring stack] with the supported configurations. You also have the option to enable xref:../monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects[monitoring for user-defined projects].
+
 include::modules/monitoring-targets-for-user-defined-projects.adoc[leveloffset=+2]
 
+The {product-title} web console provides xref:../monitoring/accessing-third-party-uis.adoc#accessing-third-party-uis[third-party integrated UIs] such as metrics, alerts, and review of monitoring dashboards. You can access these third-party user interfaces (UIs) either through the web console or the CLI. You can use the **Administrator** or ** Developer** perspective in the {product-title} monitoring dashboard to perform the following tasks:
```

Comment:
I get a bit confused here. Is this section supposed to be part of modules/monitoring-targets-for-user-defined-projects.adoc? Because it appears that way in the HTML, under the same title. This text appears to be mostly about the console, not user-defined targets.

Reply:
I can see the benefit to the reader to adding this planning content to summarise the metrics, alerts and dashboard capabilities, plus the troubleshooting monitoring topic, with links to the associated pages.

You could create a new section for this, outside of the context of monitoring targets for user-defined projects, because that is a separate topic.

Reply:
done